sdk/src/main/java/io/dapr/actors/communication/IActorResponseMessageBody.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ */
+package io.dapr.actors.communication;
+
+import java.lang.reflect.Type;
+
+/**
+ * Defines the interface that must be implemented to provide Request Message
+ * Body for remoting requests . This contains all the parameters remoting method
+ * has.
+ *
+ */
+public interface IActorResponseMessageBody {
```

Comment:
https://azure.github.io/azure-sdk/java_design.html#model-classes

Reply:
This interface is not needed for non remoting invocations.